dspace-api/src/main/java/org/dspace/workflowbasic/BasicWorkflowItem.java

```diff
@@ -183,4 +187,33 @@ public boolean isPublishedBefore() {
     public void setPublishedBefore(boolean b) {
         this.publishedBefore = b;
     }
+
+    @Override
+    public void update() throws SQLException, AuthorizeException {
+
+        Context context = null;
+        try {
+            context = new Context();
+            WorkflowServiceFactory.getInstance().getWorkflowItemService().update(context, this);
+        } finally {
+            if (context != null && context.isValid()) {
+                context.abort();
+            }
+        }
+    }
+
+    @Override
+    public String getTypeText() {
+        return "workflowitem";
```

Comment:
Should this be in `Constants`?

Reply:
done https://github.com/DSpace/DSpace/pull/2312/files#diff-537f4a26a5f610d5b958104155e9d36aR207